static/styles/zulip.css

```diff
@@ -2355,7 +2355,7 @@ div.emoji {
 
 img.emoji {
     /* This is the right offset for <img> style emoji */
-    top: 3px;
+    top: 6px;
 }
 
 .emoji.realm-emoji {
```

Comment:
@timabbott This seems to be a dead code. Shall I remove it and submit it as a commit in this PR itself.